Octokit.Reactive/Clients/IObservableRepositoryCommentsClient.cs

```diff
@@ -89,5 +89,16 @@ public interface IObservableRepositoryCommentsClient
         /// <param name="number">The comment id</param>
         /// <returns></returns>
         IObservable<Unit> Delete(string owner, string name, int number);
+
+        /// <summary>
+        /// Creates a reaction for specified Commit Comment
+        /// </summary>
+        /// <remarks>http://developer.github.com/v3/repos/comments/#create-reaction-for-a-commit-comment</remarks>
```

Comment:
This URL doesnt go anywhere.  Is it meant to be /reactions/ rather than /repos/ ?